django/core/management/commands/squashmigrations.py

```diff
@@ -184,10 +177,7 @@ def handle(self, **options):
         # need to feed their replaces into ours
         replaces = []
         for migration in migrations_to_squash:
-            if migration.replaces:
-                replaces.extend(migration.replaces)
-            else:
-                replaces.append((migration.app_label, migration.name))
+            replaces.append((migration.app_label, migration.name))
```

Comment:
```suggestion
        replaces = [(m.app_label, m.name) for m in migrations_to_squash]
```
We can use a list comprehension here, remove the row above.